frontend/public/components/utils/cog.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-undef */
```

Comment:
Does this need to be disabled for the entire file?

Reply:
It's just cleaner; the TypeScript config catches unused vars and parameters in `.tsx` files. Eventually we should switch to `tslint` so we don't need this hack.